Composer/packages/client/src/pages/publish/publishStatusList.tsx

```diff
@@ -8,25 +8,24 @@ import {
   DetailsListLayoutMode,
   SelectionMode,
   IColumn,
-  IGroup,
   CheckboxVisibility,
 } from 'office-ui-fabric-react/lib/DetailsList';
 import { Sticky, StickyPositionType } from 'office-ui-fabric-react/lib/Sticky';
 import { TooltipHost } from 'office-ui-fabric-react/lib/Tooltip';
-import { Selection } from 'office-ui-fabric-react/lib/DetailsList';
 import { Icon } from 'office-ui-fabric-react/lib/Icon';
 import { Link } from 'office-ui-fabric-react/lib/Link';
 import { Spinner, SpinnerSize } from 'office-ui-fabric-react/lib/Spinner';
 import moment from 'moment';
-import { useMemo, useState, useEffect } from 'react';
+import { useState } from 'react';
 import formatMessage from 'format-message';
+import { ActionButton } from 'office-ui-fabric-react/lib/Button';
 
 import { listRoot, tableView, detailList } from './styles';
 
 export interface IStatusListProps {
   items: IStatus[];
-  groups: IGroup[];
-  onItemClick: (item: IStatus | null) => void;
+  onLogClick: (item: IStatus | null) => void;
```

Comment:
item? Status

Reply:
yep

Reply:
I meant to make the arg optional instead of ```item: Status | null``` => ```item ?: Status```

Reply:
I have replaced it to `item: Status`